i18next.js

```diff
@@ -162,8 +162,7 @@
     }
     return matched;
   };
-  const deepFind = function (obj, path) {
-    let keySeparator = arguments.length > 2 && arguments[2] !== undefined ? arguments[2] : '.';
+  const deepFind = (obj, path, keySeparator = '.') => {
```

Comment:
@ripecosta I hope this type of stuff will not break in some user land scenarios, for the next release

Reply:
Same. TBH I was surprised `npm run build` producing this change because it seems seems to have been introduced way back in v20 ([commit](https://github.com/i18next/i18next/commit/f28b338e1b1f33888d99a6d87e8009000f2a303d#diff-e6acb643b256908cbf07c9a9c2dc7456fdba835f0f20ba343f4afc6361aee627R4-R24)) and later moved to another file in v22 ([commit](https://github.com/i18next/i18next/commit/ac4b6701c3ce9596e4eb88f5d774ca66f05d71fb#diff-3274f1a37032fb0ae4e2823def0007c634e869ae0dfc304ff6a12c36513c3a52R163)). I'm not sure where the deepFind with two params came from.